secure_redis/decorators.py

```diff
@@ -0,0 +1,159 @@
+from __future__ import (absolute_import, division, print_function,
```

Comment:
@GhalebKhaled Can you please make sure that we actually need all those to be imported here? Also, this import can fit in one line, no need for wrapping.